File: Javascript/linearsearch.js
// Linear Search Algorithm in Javascript 

function linearSearch(arr, x) {
    for (var i = 0; i < arr.length; i++) {
        if (arr[i] == x) {
            return i;
        }
    }
    return -1;
}

var arr = [2, 3, 4, 10, 40];
var x = 10;
var result = linearSearch(arr, x);
if (result == -1) {
    console.log("Element is not present in array");
}
else {
    console.log("Element is present at index " + result);
}

// Output:
// Element is present at index 3

File: C/linearsearch.c
// Linear Search Algorithm in C

#include <stdio.h>

int LinearSearch(int arr[], int n, int x)
{
    int i;
    for (i = 0; i < n; i++)
    {
        if (arr[i] == x)
        {
            return i;
        }
    }

    return -1;
}

int main()
{
    int arr[5] = {2,
                  3,
                  4,
                  10,
                  40};
    int x = 10;
    int result = LinearSearch(arr, 5, x);
    printf("%d", result);

    return 0;
}

// Output

// 3